lametro/templates/lametro/contact.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+{% block title %}Contact Us{% endblock %}
+
+{% block full_content %}
+<div class="container-fluid">
+  <div class="col-sm-12">
+  <div class="row-fluid">
+    <br />
+    <div class="col-sm-9 no-pad-mobile">
+      <h1>Contact Us</h1>
+    </div>
+    <div class="col-sm-9 no-pad-mobile">
+      <p>For questions about or comments for the Metro Board, please contact <a href="mailto:boardclerk@metro.net">boardclerk@metro.net</a>. Written public comment can be mailed to: Board Administration, 1 Gateway Plaza, Mail Stop 99-3-1, Los Angeles, CA 90012.</p>
```

Comment:
What do you think about breaking the address into a few lines here and on the about page, to make the text a bit more scannable?